health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AppHelpTutorial.js

```
@@ -0,0 +1,205 @@
+const AppHelpDrawer = ({ closing, handleClose, module, pathVar }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={`tutorial-overlay ${closing ? "fade-out" : "fade-in"}`} onClick={handleClose}>
+      <div
+        className={`tutorial-drawer ${closing ? "slide-out" : "slide-in"}`}
+        onClick={(e) => e.stopPropagation()} // prevent closing when clicking inside drawer
+      >
+        <div className="tutorial-header">
+          <span style={{ display: "flex", justifyContent: "center", gap: "0.5rem" }}>
+            <SVG.Input></SVG.Input>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)